pkg/api/types.go

```diff
@@ -1518,6 +1518,9 @@ type PodSecurityContext struct {
 	//
 	// If unset, the Kubelet will not modify the ownership and permissions of any volume.
 	FSGroup *int64 `json:"fsGroup,omitempty"`
+	// The seccomp profile that containers in the pod will run with.  Any container
+	// security context not defining its own profile will inherit the pod level profile.
+	SeccompProfile *string `json:"seccompProfile,omitempty"`
```

Comment:
will it always for every use case?  Mentioned earlier, but so far we've made the distinction between empty and not set.  If the field is not set then we can default it.  If it is purposefully set to empty then we need to be able to make decisions based on that later down the road.